sw/device/tests/spi_host_macronix_flash_test.c

```diff
@@ -0,0 +1,115 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+#include <assert.h>
+
+#include "spi_host_flash_test_impl.h"
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_pinmux.h"
+#include "sw/device/lib/dif/dif_spi_host.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/runtime/print.h"
+#include "sw/device/lib/testing/spi_device_testutils.h"
+#include "sw/device/lib/testing/spi_flash_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+static_assert(__BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__,
+              "This test assumes the target platform is little endian.");
+
+OTTF_DEFINE_TEST_CONFIG();
+
+static status_t pinmux_connect_spi_host1_to_bob(const dif_pinmux_t *pinmux) {
```

Comment:
Should this function be relocated to one of the testutils libraries?  Configuring the pinmux this way seems like it shouldn't be unique to the Macronix flash part.

Reply:
I considered that, but I can't think of a way to make it generic enough. The `csb` pin will be different for each flash, and we also have a test that uses the `spi0`. Maybe we could have a function called `pinmux_connect_spi1` which would only receive the `csb` pin as an argument?

Reply:
Done